File: src/lugbygame/Status.java
package lugbygame;

//current status
public class Status {
	int ballOwner;
	int turn;
	// 플레이어 위치정보
	// -1 : 공 //0~5 : 우리팀 //6~11 : 상대팀
	Unit[] mine;
	Unit[] enemy;
	Unit ball;

	public Status() {
		mine = new Unit[6];
		enemy = new Unit[6];
	}

	public void setStatus(Unit[] info, int ballOwner, int turn) {
		ball = info[0];
		System.arraycopy(info, 1, mine, 0, 6);
		System.arraycopy(info, 7, enemy, 0, 6);

		this.ballOwner = ballOwner;
		
		this.turn = turn;
	}
}

File: lucby_edit/Status.py
class Status:
	ballOwner=0
	turn=0
	mine=[]
	enemy=[]
	ball=None
	
	def __init__(self):
		self.mine=[0]*6
		self.enemy=[0]*6
	
	def setStatus(self, info, ballOwner, turn):
		self.ball=info[0]
		for i in range(0, 6):
			self.mine[i] = info[i+1]
		for i in range(0, 6):
			self.enemy[i] = info[i+7]
		self.ballOwner=ballOwner
		self.turn=turn
